lib/ansible/modules/packaging/os/zypper_repository.py

```diff
@@ -53,12 +53,11 @@
         required: false
         default: none
         description:
-            - A description of the repository
+            - A description (Name) of the repository
     disable_gpg_check:
         description:
             - Whether to disable GPG signature checking of
-              all packages. Has an effect only if state is
-              I(present).
+              all packages. Has an effect only if I(state=present).
```

Comment:
Ansible prefers native yaml syntax over the `a=b` notation, so I would keep the old version here.